docs/gettinginvolved/infrastructure/release-process.rst

```diff
@@ -409,27 +409,17 @@ Post-Release Review Process
 
      XXX: This step should be automated. See `FLOC-1039 <https://clusterhq.atlassian.net/browse/FLOC-1039>`_.
 
-   * The node package (``flocker-node``) should be installed on all supported platforms.
-     You can use vagrant to boot a clean Fedora 20 machine as follows:
+   * The node package (``flocker-node``) should be installed and tested on all supported platforms.
```

Comment:
Ideally this will be done by Buildbot, so please create an issue for this and link to it here.